fiftyone/utils/cvat.py

```diff
@@ -270,12 +291,23 @@ def import_annotations(
                     label_schema,
                     label_types,
                     anno_backend,
-                    anno_key,
+                    task_anno_key,
                     **kwargs,
                 )
-    finally:
-        anno_backend.delete_run(dataset, anno_key)
-        api.close()
+            except:
```

Comment:
I am okay with logging warnings for failed tasks rather than terminating, but please keep the existing code structure, eg calling `api.close()` in a `finally` block so that it always runs, even if the user ctrl + C's or something.